client/src/pages/SearchMovie/MovieDetails.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import AddMovieButton from './AddMovieButton'
+import { MovieDetailContainer } from './styled'
+
+const MovieDetails = ({ movie, revealSelected, isOpen }) => {
+  function revealDetail(title) {
```

Comment:
Since this function returns true or false, rename to `shouldRevealDetail` or `isDetailVisible`